tests/test_mongodb_namespace_restrictions.py

```diff
@@ -239,9 +288,13 @@ def test_run(self):
             self.assertGreaterEqual(v, 6)
 
         # Verify that we got 2 records with _SDC_DELETED_AT
-        self.assertEqual(2, len([x['data'] for x in records_by_stream[coll_name_1] if x['data'].get('_sdc_deleted_at')]))
-        self.assertEqual(2, len([x['data'] for x in records_by_stream[coll_name_2] if x['data'].get('_sdc_deleted_at')]))
+        self.assertEqual(2, len([x['data'] for x in records_by_stream[self.coll_name_1]
+                                 if x['data'].get('_sdc_deleted_at')]))
+        self.assertEqual(2, len([x['data'] for x in records_by_stream[self.coll_name_2]
+                                 if x['data'].get('_sdc_deleted_at')]))
         # Verify that the _id of the records sent are the same set as the
         # _ids of the documents changed
-        actual = set([ObjectId(x['data']['_id']) for x in records_by_stream[coll_name_1]]).union(set([ObjectId(x['data']['_id']) for x in records_by_stream[coll_name_2]]))
+        actual = set(
```

Comment:
Same comment from above about not doing this with set unions

Reply:
Similar update to other file